repair/row_level.cc

```diff
@@ -1095,24 +1095,32 @@ class repair_meta {
         });
     }
 
+    future<> clear_row_buf() {
+        return utils::stall_free_clear_outside_thread(_row_buf);
+    }
+
+    future<> clear_working_row_buf() {
+        return utils::stall_free_clear_outside_thread(_working_row_buf).then([this] {
+            _working_row_buf_combined_hash.clear();
```

Comment:
Will this cause a stall too?

Reply:
No. _working_row_buf_combined_hash is a single hash, not a container.